pygeo/parameterization/DVGeoMulti.py

```diff
@@ -2892,7 +2904,10 @@ def _getIntersectionSeam(self, comm, firstCall=False):
 
                 # append this new curve to the featureCurve data
                 remeshedCurves = np.vstack((remeshedCurves, newCoor))
-                remeshedCurveConn = np.vstack((remeshedCurveConn, newBarsConn))
+                remeshedCurveConnFull = np.vstack((remeshedCurveConnFull, newBarsConn))
+
+                if curveName not in self.slidingCurves:
```

Comment:
By excluding sliding curves here, they are not used as seeds for the curved-based deformation. This means that points on these curves get warped like any other point. We also still want the 'full' connectivity because that is used for projections. Is this correct? A comment or two about this would be helpful

Reply:
I will defer to @anilyil on this.